CONTRIBUTING.md

```diff
@@ -11,8 +11,13 @@ xcode-select --install
 
 Install [node.js](https://nodejs.org/) version 4 or greater
 ```bash
+brew update
```

Comment:
This is not necessary in newer releases of brew https://github.com/Homebrew/brew/pull/679 😄